ReactiveTask.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
Do we want to commit these?

Reply:
Without this we can not use New Build System in Xcode 9.4 (in Xcode the New Build System is the default so it will not be neede but an explictness will be good.